InterviewPresenter.swift

```diff
@@ -0,0 +1,51 @@
+
+import UIKit
+import Foundation
+
+protocol ProfileViewInput: AnyObject {
+    var output: ProfileViewOutput? { get set }
+    
+    func setResult(value: String)
+}
+
+protocol ProfileViewOutput {
+    func viewIsReady()
+}
+
+enum TypeCalculate: String {
+    case sum
+    case difference
+}
+
+class Presenter: ProfileViewOutput {
+    weak var view: ProfileViewInput!
+    
+    var a = Int()
+    let b: Int
+    var c = 0
+    let typeCalculate: TypeCalculate
+    
+    init(koef a: Int, b: Int, c: Int, typeCalculate: TypeCalculate) {
```

Comment:
`koef` like coefficient? it doesn't really work this way
so consider update naming